src/test/java/it/gov/pagopa/cgn/portal/facade/ProfileFacadeTest.java

```diff
@@ -191,4 +195,15 @@ void adminApproveAgreement() {
         backofficeAgreementService.approveAgreement(agreementEntity.getId());
     }
 
+    @Test
+    @Transactional
+    void Get_Profile_Expect_OK(){
```

Comment:
maybe we should add some expects to check that referents are actually saved and returned.